lib/assets/javascripts/_modules/search.js

```diff
@@ -6,35 +6,47 @@
   Modules.Search = function Search () {
     var s = this
     var $html = $('html')
-    var $tocNav
     var $searchForm
     var $searchLabel
     var $searchInput
     var $searchResults
     var $searchResultsTitle
     var $searchResultsWrapper
-    var $searchResultsClose
+    var $searchHelp
     var results
     var query
-    var queryTimer
     var maxSearchEntries = 20
 
     this.start = function start ($element) {
       $searchForm = $element.find('form')
       $searchInput = $element.find('#search')
       $searchLabel = $element.find('.search__label')
-      $searchResultsWrapper = $element.find('.search-results')
+      $searchResultsWrapper = $('#search-results')
       $searchResults = $searchResultsWrapper.find('.search-results__content')
       $searchResultsTitle = $searchResultsWrapper.find('.search-results__title')
-      $searchResultsClose = $searchResultsWrapper.find('.search-results__close')
-      $tocNav = $('#toc')
+      $searchHelp = $('#search-help')
+
       s.downloadSearchIndex()
```

Comment:
Would it make sense to now only download the search index if we're on the search page? If the search JSON has poor cache headers (which sadly is probably quite likely) then it'll be re-requested when the user hits the search page anyway.

Reply:
Mmm, you're right, we might as well avoid the extra requests. If in future we make it easier to have better caching we can always change it back.

Reply:
Sounds good. At some point it might also make sense to try and dynamically preload the search index when the user interacts with the search box, but again that only really makes sense if we know that any caching control will allow the cached index to be re-used on the next page, otherwise we're back to loading it twice.

Reply:
I've added a separate commit to make this change, it also required a tweak to the tests.